Asustor/app.json

```diff
@@ -0,0 +1,10 @@
+{
+    "appid": "80ae543b72401945334f11915bf93263ce206986",
+    "name": "Asustor",
+    "website": "https://www.asustor.com/",
+    "license": "GNU General Public License v1.0 or later",
```

Comment:
You sure this is correct?

Reply:
No, I just used the defaults from the downloaded files and didn´t find anything for that.